### ISSUE TITLE
Mise en place d'une interface de suivi

### ISSUE THREAD

**Issue by DonovanMaillard** [lpoaura/GN2PG#85]
### Feature description

Le nombre de sources commence à se multiplier pour le Pôle Invertébrés, et il devient chronophage de savoir quelle source a été rafraichie, quand, et avec quel résultat. 

Avoir une interface de synthèse, éventuellement directement dans le module admin de GeoNature si on veut pousser le vice, serait une bonne chose pour suivre nos transmissions/intégrations de données.

**Comment by camillemonchicourt** [comment]
Oui c'est vraiment dans cette optique que l'on avait fait développer une première Admin de GN2PG : https://github.com/lpoaura/GN2PG/pull/39

Dans un premier temps limitée à la consultation des logs car déjà existants, mais idéalement prévue pour pouvoir suivre les intégrations de données.

Cependant on peut se demander si il faut vraiment enrichir ça au niveau de GN2PG, ou si ce n'est un besoin plus global au niveau de GroNature de pouvoir avoir un outil de suivi de ses données, leur intégration (saisir, import manuel, import automatique...), leur diffusion, etc...

Je penche plus sur cette deuxième option, mais pourquoi pas déjà enrichir le suivi côté GN2PG qui serait plus simple pour commencer.

**Comment by DonovanMaillard** [comment]
oui, là l'idée serait d'avoir un premier outil dans les prochains mois car les échanges se multiplient. 

A voir si cette interface se fait, si on peut toujours multiplier n fichiers de configuration et n schémas gn2pg comme je le fais aujourd'hui. Ca peut compliquer les choses, il ne faut pas que cette manière de l'utiliser (tout a fait fonctionnelle) ne vienne alourdir la gestion de l'outil par ailleurs.

**Comment by lpojgc** [comment]
oui ça rejoint https://github.com/lpoaura/GN2PG/issues/77

**Comment by jpm-cbna** [comment]
> oui ça rejoint #77

Cela rejoint également nos retours #53 concernant l'amélioration de Gn2Pg pour facilité le suivi des erreurs et la reprise sur échec.
